pkg/config/validation/validation.go

```diff
@@ -1235,6 +1235,23 @@ func ValidateProtocolDetectionTimeout(timeout *types.Duration) error {
 	return nil
 }
 
+// ValidateMaxServerConnectionAge validate positive duration
+func ValidateMaxServerConnectionAge(in time.Duration) error {
```

Comment:
This is not the right place. The flag should be validated during bootstrap under `pilot/cmd`

Reply:
This is a common validation, calling now from `pilot/cmd`

Reply:
The package is mostly used mostly for admission webhook to validate the API, for flags they belong to process startup

Reply:
We may need to validate other flags in the future